perf/roles/trex/defaults/main.yml

```diff
@@ -2,5 +2,8 @@
 
 trex_repo: https://github.com/rh-nfv-int/trex-core
 trex_tag: rhosp-playbooks
-trex_dir: /tmp/trex-core
 trex_opts: ''
+trex_release_url: "https://trex-tgn.cisco.com/trex/release/latest"
```

Comment:
This url is a symlink to whatever the latest version is. You are effectively not downloading the version you are expecting.

`trex_release_url` should be constructed in the playbook when downloading: `https://trex-tgn.cisco.com/trex/release/{{trex_version}}.tar.gz`

Reply:
Updated ..